site/verifyemail.html.jet

```diff
@@ -0,0 +1,18 @@
+{{extends "templates/application/application.jet"}}
+
+{{block head()}}
+  {{yield seo(title=i18n("signup_page_header"), index=false)}}
```

Comment:
Probably want to change that signup_page_header to something new.

Reply:
It's meant to be the same. When the user has the verify email toggle on then this is the first step of creating a new account.